subsys/debug/gpio_frontend/Kconfig

```diff
@@ -0,0 +1,31 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+if LOG_FRONTEND
+config LOG_FRONTEND_GPIO
+	bool "Enable GPIO frontend"
+	help
+	  When enabled frontend is using GPIO pins to output logs.
+
+if LOG_FRONTEND_GPIO
+config LOG_FRONTEND_PORT_NUMBER
+	int "GPIO port"
+	default 0
+	range 0 1 if HAS_HW_NRF_GPIO1
+	range 0 0
+
+config LOG_FRONTEND_LSB_PIN_NUMBER
```

Comment:
I assume that I have to have 9 lines of GPIO located "nearby". Could we update the code to select arbitrary GPIO lines? This would allow to use this backend on other boards than DK. Ideally there should be also options using 3 and 5 GPIOs (2-bits, 4-bits per clock).